src/components/TextField.tsx

```diff
@@ -9,11 +9,17 @@ import {
   FormHelperText,
   FormErrorMessage,
   Divider,
+  IconButton,
+  InputLeftElement,
+  Spinner,
 } from '@chakra-ui/react';
+import { CopyIcon, CheckIcon, WarningIcon } from '@chakra-ui/icons';
+import CopyToClipboard from 'react-copy-to-clipboard';
```

Comment:
Why do we need a library for this? Can use `navigator.clipboard.writeText("boo")`

Reply:
Whoops it's the same one used in platform portal, and now I'm wondering why that library needed to exist in the first place 🤔

Reply:
I guess there probably wasn't a good web api for it at some point, but I think it's redundant now